addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -54,11 +54,26 @@
     <div class="o_newsletter_popup o_snippet_invisible" data-list-id="0"/>
 </template>
 
+<template id="s_newsletter_subscribe_popup_content" name="Newsletter Popup Content">
+    <section class="oe_img_bg pt88 pb64" style="background-image: url('/web/image/website.s_cover_default_image'); background-position: 0 100%;">
+        <div class="container s_allow_columns">
+            <h1 style="text-align: center;">Always <b>First</b>.</h1>
+            <p style="text-align: center;">Be the first to find out all the latest news,<br/> products, and trends.</p>
+        </div>
+    </section>
+    <section>
+        <div class="container">
+            <div class="row s_nb_column_fixed">
+                <div class="col-lg-8 offset-lg-2 pt32 pb32">
+                    <t t-snippet-call="website_mass_mailing.s_newsletter_subscribe_form"/>
+                </div>
+            </div>
+        </div>
+    </section>
```

Comment:
Mmh, the point of using a base snippet for that popup was that you could re-add it if you removed it from the popup... but I guess the user can just CTRL-Z or redrop the popup snippet. Should we remove the s_newsletter_block snippet though? cc @sbuhl 

However, if we do that, the `section` elements here should be standard snippet (so appropriate s_ classes and data-snippet attribute), otherwise we cannot deprecate them ever ;)